egs/aishell2/s5/local/chain/tuning/run_tdnn_1b.sh

```diff
@@ -1,24 +1,30 @@
 #!/bin/bash
 
-# _1d is as _1c, but with dropout schedule added, referenced from wsj
+# _1b is as _1a, but with pitch feats, i-vector and dropout schedule added, referenced from wsj
+# this script is for 'normal' mode
```

Comment:
it would only run correctly in 'normal' mode due to feature dimension. I think it has been indicated in run.sh, where setting different modes leads to different scripts. But of course I can make changes to make it potentially simpler.